testutil/singlevm/verify.sh

```diff
@@ -137,3 +137,37 @@ fi
 
 "$ciao_gobin"/ciao-cli instance list
 "$ciao_gobin"/ciao-cli instance delete --all
+
+#Wait around a bit as instance delete is asynchronous
+retry=0
+ciao_networks=0
+until [ $retry -ge 6 ]
+do
+	#Verify that there are no ciao related artifacts left behind
+	ciao_networks=`sudo docker network ls --filter driver=ciao -q | wc -l`
+
+	if [ $ciao_networks -eq 0 ]
+	then
+		break
+	fi
+	sleep 1
```

Comment:
It might be better to wait until the instance deleted event is received, but this isn't currently working, see #776.